stacks-signer/src/signer.rs

```diff
@@ -1453,3 +1472,41 @@ impl Signer {
         Ok(())
     }
 }
+
+fn encrypt(private_key: &Scalar, msg: &[u8]) -> Vec<u8> {
+    wsts::util::encrypt(&derive_encryption_key(private_key), msg, &mut OsRng)
+        .expect("Failed to encrypt message")
+}
+
+fn decrypt(private_key: &Scalar, encrypted_msg: &[u8]) -> Vec<u8> {
+    wsts::util::decrypt(&derive_encryption_key(private_key), encrypted_msg)
+        .expect("Failed to decrypt message")
+}
+
+fn derive_encryption_key(private_key: &Scalar) -> [u8; 32] {
+    let mut hasher = Sha256::new();
```

Comment:
You can safely use `Sha512Trunc256Sum` here from `stacks_common`.

Reply:
Thanks, will do.

Reply:
Done.